src/wallet/feebumper.cpp

```diff
@@ -111,22 +112,28 @@ Result CreateTransaction(const CWallet* wallet, const uint256& txid, const CCoin
         return Result::INVALID_ADDRESS_OR_KEY;
     }
     const CWalletTx& wtx = it->second;
+    if (reduce_output < -1 || reduce_output >= int64_t(wtx.tx->vout.size())) {
+        errors.push_back(strprintf("Change output out of bounds [-1..%zu]", wtx.tx->vout.size()-1));
```

Comment:
The default is `-1` to mean `auto-select`. How would you propose that is expressed otherwise? As an additional `bool explicit_reduce_output` feels messy.

Reply:
@kallewoof Just change the error message (and prevent -1 from being passed in from the user)

Reply:
@luke-jr I can't distinguish from user passing -1 and from the default (which means 'auto-pick' like the current behavior). I can change the message to say `0..`, though. And it already prevents user from passing -1 in the RPC command itself.